projects/plugins/jetpack/extensions/blocks/ai-assistant/components/ai-assistant-controls/index.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * External dependencies
+ */
+import {
+	MenuItem,
+	MenuGroup,
+	ToolbarDropdownMenu,
+	CustomSelectControl,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { post, termDescription } from '@wordpress/icons';
+import React from 'react';
+/**
+ * Internal dependencies
+ */
+import AIAssistantIcon from '../../icons/ai-assistant';
+import './style.scss';
+
+// Quick edits option: "Correct spelling and grammar"
+const QUICK_EDIT_KEY_CORRECT_SPELLING = 'correct-spelling' as const;
```

Comment:
TIL about the `as const` :clap:

Reply:
it's a way to combine constants with types